browser-test/src/admin_application_statuses.test.ts

```diff
@@ -161,7 +161,34 @@ describe('view program statuses', () => {
       const {adminPrograms} = ctx
       await adminPrograms.editNote('Some note content')
       await adminPrograms.expectNoteUpdatedToast()
-      // TODO(#3264): Assert that the note has been updated.
+    })
+
+    it('shows the current note content', async () => {
+      const {adminPrograms} = ctx
+      const noteText = 'Some note content'
+      await adminPrograms.editNote(noteText)
+      await adminPrograms.expectNoteUpdatedToast()
+
+      // Reload the note editor.
+      await adminPrograms.viewApplications(programWithStatusesName)
+      await adminPrograms.viewApplicationForApplicant('Guest')
+
+      expect(await adminPrograms.getNoteContent()).toBe(noteText)
+    })
+
+    it('allows updating a note', async () => {
```

Comment:
Not blocking: I'm not sure having two distinct tests here really has more value than combining them into a single test that:
  1) Edits the note
  2) Confirms that the displayed content matches the edit
  3) Edit the note again
  4) Confirms that the updated content matches the edit

As it stands, the first test is already testing the "update" case as part of test setup.

Reply:
Your argument is definitely valid. My main reaction was that I saw the two tests that had almost all of the same code and it wasn't immediately obvious to me what the difference between the two of them was until I read closer.